builtin-doc/coq-builtin.elpi

```diff
@@ -1458,7 +1458,7 @@ external pred coq.unify-eq i:term, i:term, o:diagnostic.
 % if they are types)
 external pred coq.unify-leq i:term, i:term, o:diagnostic.
 
-% [coq.elaborate-skeleton T ETy E Diagnostic] elabotares T against the
+% [coq.elaborate-skeleton T ETy E Diagnostic] elaborates T against the
```

Comment:
This file is generated from `src/coq_elpi_builtins.ml` so the typo should primarily be fixed there.